libraries/AP_HAL_SITL/HAL_SITL_Class.cpp

```diff
@@ -80,13 +80,17 @@ static char *new_argv[100];
 /*
   save watchdog data
  */
-static void watchdog_save(const uint32_t *data, uint32_t nwords)
+static bool watchdog_save(const uint32_t *data, uint32_t nwords)
```

Comment:
is anyone looking at this result?

Reply:
just the *$&! compiler, which insists on not ignoring return from write()